example/example.go

```diff
@@ -1,29 +1,27 @@
 package main
 
 import (
-	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/kms"
-	"github.com/dgrijalva/jwt-go"
-	"github.com/matelang/jwt-go-aws-kms/v2/jwtkms"
+	"context"
 	"log"
 	"time"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/kms"
+	"github.com/dgrijalva/jwt-go"
+	"github.com/matelang/jwt-go-aws-kms/v2/jwtkms"
 )
 
-const keyId = "aa2f90bf-f09f-42b7-b4f3-2083bd00f9ad"
+const keyID = "aa2f90bf-f09f-42b7-b4f3-2083bd00f9ad"
 
 func main() {
-	sess := session.Must(session.NewSessionWithOptions(session.Options{
-		SharedConfigState: session.SharedConfigEnable,
-	}))
-	svc := kms.New(sess)
-
-	keyConfig := &jwtkms.KmsConfig{
-		Svc:      svc,
-		KmsKeyId: keyId,
+	awsCfg, err := config.LoadDefaultConfig(context.TODO(),
```

Comment:
```suggestion
	awsCfg, err := config.LoadDefaultConfig(context.Background(),
```
that should be ok in main